deploy/helm/flightctl/templates/flightctl-gateway.yaml

```diff
@@ -0,0 +1,48 @@
+{{ if and (.Values.api.enabled) (eq (include "flightctl.getServiceExposeMethod" .) "gateway") }}
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: flightctl-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  gatewayClassName: {{ .Values.global.gatewayClass }}
+  listeners:
+  {{- if and (eq .Values.global.target "standalone") (eq .Values.global.auth.type "builtin") }}
+  - name: keycloak
+    hostname: auth.{{ include "flightctl.getBaseDomain" . }}
+    {{- if .Values.global.baseDomainTls.cert }}
+    port: {{ .Values.global.gatewayPorts.tls }}
+    protocol: HTTPS
+    tls:
+      mode: Terminate
+      certificateRefs:
+      - kind: Secret
+        name: gateway-cert
+    {{- else }}
+    port: {{ .Values.global.gatewayPorts.http }}
+    protocol: HTTP
+    {{- end }}
+  {{- end }}
+  {{- if .Values.ui.enabled }}
+  - name: ui
```

Comment:
would it make sense to have the UI have it's own gateway?

Reply:
I tried but If I move the UI specific bits to a separate gateway, it doesn't work and I dont see any error anywhere :/